### DISCUSSION TITLE
My KDE disappered

### DISCUSSION THREAD

**Discussion by VicWang17** [prasanthrangan/hyprdots#808] [Q&A]
I used to use the KDE version of Manjaro, based on which I installed hyprland and this dots. However, after all the configuration was completed, every time I started up my laptop, the screen would be black, so I needed to switch to tty2 to manually open hyprland, and at the same time my kde was missing, could I still find my original kde?

**Comment by kRHYME7** [comment]
hello, what display manager were you using before installing Hyprdots?

**Reply by VicWang17** [reply]
> hello, what display manager were you using before installing Hyprdots?
> 

I think it should be sddm. i'm not set it before, i use Manjaro KDE

**Reply by kRHYME7** [reply]
So sddm did not show at all after boot? 

``` systemctl status sddm.service ``` What's the output of this

**Reply by VicWang17** [reply]
![IMG20240130142250.jpg](https://github.com/prasanthrangan/hyprdots/assets/44319845/891e5ea6-5f3d-427c-b66e-9c21f260ba8d)

it looks like this....tty1 is black, i can start  my hyprland on tty2

**Reply by kRHYME7** [reply]
I think sddm broke.  

Maybe unrelated, this system use Nvidia?

**Reply by VicWang17** [reply]
> I think sddm broke.  
> 
> Maybe unrelated, this system use Nvidia? 

yes...i think nvidia caused this problem, now i'm trying to resetup the Linux

**Reply by kRHYME7** [reply]
In case the installation of hyprdots gives this issue maybe manually deal with Nvidia? 

Edit  ./install.sh you can remove arguments for Nvidia so you can continue.
or Edit ``` ~/Hyprdots/Scripts/global_fn.sh    ``` and make all return values to 1 inside 
the nvidia detect so that script thinks you're not using nvidia
```
nvidia_detect()
{
    if [ `lspci -k | grep -A 2 -E "(VGA|3D)" | grep -i nvidia | wc -l` -gt 0 ]
    then
        #echo "nvidia card detected...  but will return 0"
        return 1 
    else
        #echo "nvidia card not detected..."
        return 1
    fi
}
```
